test/plugin/mock-collector/src/main/java/org/apache/skywalking/plugin/test/mockcollector/Main.java

```diff
@@ -47,6 +55,18 @@ public static void main(String[] args) throws Exception {
         String contextPath = "/";
         ServletContextHandler servletContextHandler = new ServletContextHandler(ServletContextHandler.NO_SESSIONS);
         servletContextHandler.setContextPath(contextPath);
+        servletContextHandler.addServlet(new ServletHolder(new HttpServlet() {
+            @Override
+            protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+                if (ValidateData.INSTANCE.getRegistryItem().getApplications().isEmpty()) {
+                    resp.setStatus(500);
```

Comment:
Is there a case where there are multiple ```agent``` in one test case.

Reply:
Mutiple agent is possible, but let's avoid it if possible.